### PR TITLE
Run isort at pandas/tests/io

### PR DESCRIPTION
**Pull request by thoo**
- [ ] xref #23334
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by jreback** [comment]
merge master

**Comment by jreback** [comment]
need a merge master

**Comment by jreback** [comment]
can you merge again and something failng

File: setup.cfg [hunk 1]
```diff
@@ -137,52 +137,43 @@ skip=
...
+    pandas/tests/indexes/timedeltas/test_construction.py,
+    pandas/tests/indexes/timedeltas/test_setops.py,
+    pandas/tests/indexes/timedeltas/test_timedelta.py,
+    pandas/tests/indexes/timedeltas/test_tools.py,
+    pandas/tests/indexes/timedeltas/test_arithmetic.py,
+    pandas/tests/indexes/timedeltas/test_astype.py,
+    pandas/tests/indexes/timedeltas/test_scalar_compat.py,
+    pandas/tests/indexes/timedeltas/test_partial_slicing.py,
+    pandas/tests/indexes/timedeltas/test_timedelta_range.py,
+    pandas/tests/indexes/timedeltas/test_ops.py,
```
Comment (datapythonista) [inline setup.cfg]: why are we adding all these back? they should be ok if they are not in the exclude list, right?

**Comment by jreback** [comment]
@thoo can you see if can get passing

File: pandas/io/formats/style.py [hunk 2]
```diff
@@ -2,7 +2,11 @@
 Module for applying conditional formatting to
 DataFrames and Series.
 """
-from collections import MutableMapping, defaultdict
+try:
```
Comment (jreback) [inline pandas/io/formats/style.py]: you can just change this to use ``is_dict_like`` instead I think

Comment (thoo) [reply pandas/io/formats/style.py]: I am not following here. `is_dict_like` instead of ??

Comment (thoo) [reply pandas/io/formats/style.py]: Got it. Done. It is much cleaner.

**Comment by jreback** [comment]
lgtm. ping on green.

**Comment by jreback** [comment]
thanks @thoo

**Pull request merged**